### PR TITLE
Buildfix - fix datalisteditor

### PR DESCRIPTION
**Pull request by Tonypz**

### CODE REVIEW

File: src/form/editors/impl/datalist/views/ButtonView.ts [hunk 1]
```diff
@@ -40,10 +51,24 @@ export default TextEditorView.extend({
         }
     },
 
-    events: false,
+    events() {
+        const events = {
```
Comment (stasguryev) [inline src/form/editors/impl/datalist/views/ButtonView.ts]: if (!this.options.isAutocompleteMode} return {}

File: src/form/editors/DatalistEditorView.ts [hunk 2]
```diff
@@ -540,13 +550,21 @@ export default (formRepository.editors.Datalist = BaseEditorView.extend({
         this.debouncedFetchUpdateFilter(undefined, { isSearch: true });
     },
 
-    __fetchUpdateFilter(text = this.__getInputValue(), {
+    __getTextInput() {
```
Comment (stasguryev) [inline src/form/editors/DatalistEditorView.ts]: ?

File: src/form/editors/DatalistEditorView.ts [hunk 3]
```diff
@@ -524,11 +532,13 @@ export default (formRepository.editors.Datalist = BaseEditorView.extend({
...
     __clearSearch() {
-        this.__setInputValue('');
+        if (!this.options.isAutocompleteMode) {
+            this.__setInputValue('');
+        }
         this.__fetchUpdateFilter('', { open: false, isSearch: true });
     },
 
-    __getInputValue(): string {
+    __getDropdownInputValue(): string {
```
Comment (stasguryev) [inline src/form/editors/DatalistEditorView.ts]: !

File: src/form/editors/impl/datalist/views/ButtonView.ts (hunk 1 above)
Comment (Tonypz) [reply src/form/editors/impl/datalist/views/ButtonView.ts]: done

File: src/form/editors/DatalistEditorView.ts (hunk 2 above)
Comment (Tonypz) [reply src/form/editors/DatalistEditorView.ts]: done

File: src/form/editors/DatalistEditorView.ts (hunk 3 above)
Comment (Tonypz) [reply src/form/editors/DatalistEditorView.ts]: done

**Pull request closed without merging**